e2e/atlas/dbusers_test.go

```diff
@@ -82,6 +82,24 @@ func TestDBUsers(t *testing.T) {
 		}
 	})
 
+	t.Run("Describe", func(t *testing.T) {
```

Comment:
this seems to be failing on evergreen, something about the auth db not being valid

Reply:
ah nice spot, looking